readme.md

```diff
@@ -14,6 +14,12 @@
 
 - [db-prices](https://github.com/juliuste/db-prices/) – "JavaScript client for the DB Sparpreise API."
 
+## Running
+```
+npm start
+# go to http://localhost:3009/
```

Comment:
I think `3009` is a rather unusual default port. What about changing it to `3000`?

Reply:
Wouldn't `3000` be rather unusual as well? Compared to `8000` or `8080`

Reply:
`3000` is the common port for dev environments. `8000` and `8080` are often used as alternative ports for (production) HTTP on `80`, such as when adding a WebSocket server or backup service.

Reply:
Yolo.